pmm-app/src/pmm-qan/panel/components/Filters/Filters.tsx

```diff
@@ -11,22 +11,30 @@ import { useTheme } from '@grafana/ui';
 import { CheckboxGroup } from './components/CheckboxGroup/CheckboxGroup';
 import { FILTERS_BODY_HEIGHT, FILTERS_GROUPS } from './Filters.constants';
 import { getSelectedCheckboxes } from './Filters.tools';
-import { FiltersContainerProps } from './Filters.types';
 import { getStyles } from './Filters.styles';
 import { useFilters } from './hooks/useFilters';
 import { useInitialFilterValues } from './hooks/useInitialFilterValues';
 import { useFiltersContainerHeight } from './hooks/useFiltersContainerHeight';
 import { Messages } from './Filters.messages';
 
-export const FiltersContainer: FC<FiltersContainerProps> = ({ filters, disabled, rawTime }) => {
+export const Filters: FC = () => {
   const theme = useTheme();
   const styles = getStyles(theme);
 
+  const {
+    contextActions,
+    panelState: { loadingDetails, rawTime },
+  } = useContext(QueryAnalyticsProvider);
+
+  const { filters, loading } = useFilters();
+  const initialValues = useInitialFilterValues();
+
   const filtersWrapperRef = useRef<HTMLDivElement>(null);
 
   const height = useFiltersContainerHeight(FILTERS_BODY_HEIGHT, filtersWrapperRef);
-  const [filter, setFilter] = useState('');
   const [showAll, showSetAll] = useState(true);
+  const [filter, setFilter] = useState('');
+
```

Comment:
IMO this white space can be remove since there is another declaration after.

Reply:
good point, I tried to separate different declaration but without whitespaces, it's more readable